src/syscheckd/db/include/db_statements.hpp

```diff
@@ -68,3 +71,137 @@ constexpr auto CREATE_REGISTRY_VALUE_DB_STATEMENT
     PRIMARY KEY(key_id, name)
     FOREIGN KEY (key_id) REFERENCES registry_key(id));)"
 };
+
+constexpr auto FIM_FILE_SYNC_CONFIG_STATEMENT
+{
+    R"(
+    {
+        "decoder_type":"JSON_RANGE",
+        "table":"file_entry",
+        "component":"fim_file_sync",
+        "index":"path",
+        "checksum_field":"checksum",
+        "no_data_query_json": {
+                "row_filter":"WHERE path BETWEEN '?' and '?' ORDER BY path",
+                "column_list":["*"],
+                "distinct_opt":false,
+                "order_by_opt":""
+        },
+        "count_range_query_json": {
+                "row_filter":"WHERE path BETWEEN '?' and '?' ORDER BY path",
+                "count_field_name":"count",
+                "column_list":["count(*) AS count "],
+                "distinct_opt":false,
+                "order_by_opt":""
+        },
+        "row_data_query_json": {
+                "row_filter":"WHERE path ='?'",
+                "column_list":["*"],
+                "distinct_opt":false,
+                "order_by_opt":""
+        },
+        "range_checksum_query_json": {
+                "row_filter":"WHERE path BETWEEN '?' and '?' ORDER BY path",
+                "column_list":["*"],
+                "distinct_opt":false,
+                "order_by_opt":""
+        }
+    }
+    )"
+};
+
+constexpr auto FIM_REGISTRY_SYNC_CONFIG_STATEMENT
```

Comment:
When working with tables with more than one primary key we have to set an `item_id` to run the operations, instead of the `path` directly. See an example in `PACKAGES_SYNC_CONFIG_STATEMENT` in Syscollector.

Reply:
Done, created necesary vector with the primary keys too.